controller/test-files/root/list/not_found.res.payload.golden.json

```diff
@@ -0,0 +1,10 @@
+{
+  "errors": [
+    {
+      "code": "not_found",
+      "detail": "file with id 'swagger.json' not found",
```

Comment:
This should be `file with name` or just `file.. `

Reply:
I guess, this text is produced by `errrors.NewNotFoundError("file", "swagger.json").String()`. Let's live with it.

Reply:
Perhaps when we do some work on our errors we can change this